src/components/CvToggle/CvToggle-Skeleton.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
This one is not working. BTW I don't think a skeleton toggle is a very common use case but it would be good to have it working for completeness. Here is what I see:
![image](https://github.com/carbon-design-system/carbon-components-vue/assets/7536103/e645d0eb-a8b2-4466-a517-79f89208173f)

The react component looks OK
https://v7-react.carbondesignsystem.com/?path=/story/components-toggle--skeleton

Have a look there but they have the bx--skeleton on an input element.

Reply:
Hello! In the vue2 the [Skeleton](https://vue.carbondesignsystem.com/?path=/story/components-cvtoggle--skeleton) component is the same, I created based on the current V2 version.

Do you have any special requirements for that?
<img width="1512" alt="image" src="https://github.com/carbon-design-system/carbon-components-vue/assets/112895985/c13ff030-3878-4adb-9c27-a6b7fcea4e2a">